js/stores/windowStore.js

```diff
@@ -658,8 +658,8 @@ const doAction = (action) => {
     case windowConstants.WINDOW_AUTOFILL_POPUP_HIDDEN:
     case windowConstants.WINDOW_SET_CONTEXT_MENU_DETAIL:
       if (!action.detail) {
-        if (windowState.getIn('contextMenuDetail', 'type') === 'autofill' &&
-            windowState.getIn('contextMenuDetail', 'tabId') === action.tabId) {
+        if (windowState.getIn(['contextMenuDetail', 'type']) === 'autofill' &&
+            windowState.getIn(['contextMenuDetail', 'tabId']) === action.tabId) {
```

Comment:
this is probably the reason the code above was still needed. I think we can keep this and lose the `isFocused` check